AY2021S1-CS2103-T16-3/tp
# Purge Statistics

### What this does
This PR purges the statistics piechart feature for ResiReg. After discussion with the developer team, the team agreed that the statistics piechart, in consideration of the numerous other features of ResiReg, is not well packaged as a single product with a single core principled functionality based on product design considerations.

### How to test

1. Launch the application.
2. Ensure that the Statistics tab is removed.
3. Run `bundle exec jekyll serve` at `docs`.
4. Ensure that the User Guide makes no references to the piechart panel and the Developer Guide UI component does not reflect the `PiechartPanel`.

Closes #144, #151, #152, #157, #162.
